Qualifiers/src/main/res/values-de-sw540dp-1350x1800/strings.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+  ~
+  ~  Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  ~
+  -->
+<resources>
+    <!-- Your app should work without any problem on single screen mode on Surface Duo as it works on any other
+        single screen device; so it shouldn't be any reason to have specific resources for single screen mode.
+        But for informative purposes, here is the resource folder (values-sw540dp-1350x1800 that Surface Duo uses
+        when on single screen mode (both single-portrait and single-landscape) -->
+    <string name="qualifiers_app_name">Qualifikanten</string>
```

Comment:
It can work, but in other samples we have the English names for the sample-names